odoo/models.py

```diff
@@ -3532,8 +3532,11 @@ def unlink(self):
 
             # Check if the records are used as default properties.
             refs = [f'{self._name},{id_}' for id_ in sub_ids]
-            if Property.search([('res_id', '=', False), ('value_reference', 'in', refs)], limit=1):
+            default_preperties = Property.search([('res_id', '=', False), ('value_reference', 'in', refs)])
+            if not self._context.get(MODULE_UNINSTALL_FLAG) and default_preperties:
                 raise UserError(_('Unable to delete this document because it is used as a default property'))
+            else:
+                ir_property_unlink |= default_preperties
```

Comment:
/summon @ryv-odoo

Reply:
FYI, this is used in the loading of the chart template when we are replacing a chart template by another, during the setup of the database (for instance, when you have multiple CoA for one company and the default one is not the right one). We are basically uninstalling data (old template) and installing new data.

Reply:
And I put it in another commit too